sycl/test/extensions/fpga.cpp

```diff
@@ -128,5 +128,29 @@ int main() {
     }
   }
 
+  /*Check DSP control interface*/
```

Comment:
Can this feature be tested on an FPGA emulator? if so could you add E2E tests for in in intel/llvm-test-suite?

Reply:
Sure, I can add an E2E emulator test after the header is merged in.

Reply:
The link to the PR should be added to the PR description.

Reply:
Addressed.